addons/rose/app/styles/rose/_typography.scss

```diff
@@ -12,60 +12,27 @@ html {
 }
 
 body {
+  // Used by Rose::Tabs and '/' in breadcrumbs
+  // and link list panel
   font-family: typography.$family-primary;
 
-  // This is the only time we'll explicitly use pixels.  All other sizes are
-  // expressed in rem units.
-  font-size: typography.$base-font-size * 1px;
+  // these 3 are used across the app
   line-height: typography.line-height(m);
   color: var(--stark);
   background-color: var(--subtle);
 }
 
-input,
-textarea,
-button {
-  font-family: typography.$family-primary;
-}
-
-pre,
-code {
-  font-family: typography.$family-monospace;
-}
-
-h1,
-.h1 {
-  @include type-mixins.type(h1, bold);
-
-  margin-top: sizing.rems(m);
-  margin-bottom: sizing.rems(xs);
-}
-
-h2,
-.h2 {
-  @include type-mixins.type(h2, bold);
-
-  margin-top: sizing.rems(m);
-  margin-bottom: sizing.rems(xs);
+// By default, we add top and bottom margins to all headers
+// so this class can be used to remove the margin if needed
+.remove-header-margin {
```

Comment:
In retrospect, if this is called `remove-header-margin`, shouldn't it have targeted for only header tags? Isn't this class as it is just a generic "remove margin" class?